src/main/kotlin/com/openlattice/data/storage/PostgresQueries.kt

```diff
@@ -378,8 +381,13 @@ internal fun selectEntityKeyIdsWithCurrentVersionSubquerySql(
 ): String {
     val metadataColumns = getMetadataOptions(metadataOptions, linking).joinToString(",")
 
-
-    var selectColumns = joinColumns.joinToString(",")
+    var selectColumns = joinColumns.joinToString(",") { column ->
```

Comment:
Why not include entity_set_id in group by?

Reply:
linking_id crosses multiple entity_set_ids.